services/inc/services_dynalib.h

```diff
@@ -49,6 +49,7 @@ DYNALIB_FN(services,LED_RGB_SetChangeHandler)
 DYNALIB_FN(services, log_print_direct_)
 DYNALIB_FN(services, LED_GetColor)
 //DYNALIB_FN(services, log_level_active)
```

Comment:
Sure, it's quite easy to implement, I just need to add another callback to application level that will allow to determine logging level for particular category.